lib/openhab/dsl/items/generic_item.rb

```diff
@@ -131,6 +131,23 @@ def groups
           group_names.map { |name| Groups.groups[name] }
         end
 
+        # Return the item's thing if this item is associated with a thing. If an item is bound to more than one thing,
+        # this method only returns the first thing.
+        #
+        # @return [Thing] The thing associated with this item or nil
+        def thing
+          all_things.first
+        end
+
+        # Returns all of the item's associated things.
+        #
+        # @return [Array] An array of things or an empty array
+        def all_things
+          registry = OpenHAB::Core::OSGI.service('org.openhab.core.thing.link.ItemChannelLinkRegistry')
+          channels = registry.get_bound_channels(name).to_a
+          channels.map(&:get_thing_uid).uniq.map { |tuid| things[tuid] }
```

Comment:
I think you can drop the get_ cause jruby

Reply:
:facepalm: I knew that from you saying that to me last week, and I just got tunnel vision